src/ElvSpace.js

```diff
@@ -153,15 +170,27 @@ class ElvSpace {
         contractAddress: tenantContract.address,
         abi: JSON.parse(tenantContract.abi),
         methodName: "addGroup",
-        methodArgs: [contractAdminGroup, adminGroupAddress],
+        methodArgs: [contractAdminGroup, tenantAdminGroupAddress],
         formatArguments: true,
       });
 
       if (this.debug){
-        console.log("Result set admin group", res);
+        console.log("Result set tenant admin group", res);
       }
     }
 
+    res = await this.client.CallContractMethodAndWait({
+      contractAddress: tenantContract.address,
+      abi: JSON.parse(tenantContract.abi),
+      methodName: "addGroup",
+      methodArgs: ["content_admin", contentAdminGroupAddress],
```

Comment:
We should call this 'content_admins' (plural).  Is the tenant_admins currently also plural.

Reply:
The tenant admin group on the tenant contract has always been referred as `"tenant_admin"` (without an s), so I thought it would be the best to keep it consistent. Changing it to `"tenant_admins"` would require changes on both the contract and the client + deployed contracts will break so we also need to create a method that fix them.